### DISCUSSION TITLE
Mi Band WatchFace Editor GUI

### DISCUSSION THREAD

**Discussion by Johnson070** [Johnson070/MiBand-5-watchface-editor#12] [General]
1.0.1.2 (2021-04-22)

- Bug fixes

- Add full support MiBand 6(With building a bin file yet)

- Add simple select images(Hold down the left mouse button to select, and to delete with shift)

- Animation preview(use carefully, the program uses more than 100 MB of RAM)

- (FIXED) If you have a problem with the image, try changing the scale of windows to 100%

- (FIXED) Month/Date add

- (FIXED) Add wind error and today separate temperature

Thank you for downloading the program.

If you find a bug, please write to github or telegram: @vveber

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnson070/MiBand-5-watchface-editor/releases/tag/1.0.1.2'>Mi Band WatchFace Editor GUI</a>.</em>

**Comment by Johnson070** [comment]
What should be added in the next update?

**Reply by ldicdican** [reply]
Some wiki on how to use the app, how to install on smart watchs 👍

**Reply by Johnson070** [reply]
Ok

**Reply by ldicdican** [reply]
So excited on using your app, I'm a graphic artist and will do my best to create bunch of watch faces! Thank you very much in advance, very much appreciated all your efforts on developing this wonderful app 👍

**Reply by Johnson070** [reply]
Thank you so much.  Glad to think this program helps people create watchfaces

**Reply by Johnson070** [reply]
try to search on Google how to install watch faces or I will make instructions later

**Reply by ldicdican** [reply]
Thanks, already installed Notify for Mi and it works flawlessly installing bin watch face files created by your app. Thanks a lot ! 👍
